README.md

```diff
@@ -83,6 +84,29 @@ If you have multiple JFrog platform configurations as secrets, you can use all o
 | Important: When exposing more than one JFrog configuration to the Action, you should always add the ```jf c use``` command to specify the server to use. |
 | --- |
 
+### Setting direct connection details environment variables
+You can use connection details directly by settings one of the following environment variables combinations:
+1. JF_URL (anonymous)
+2. JF_URL + JF_USER + JF_PASSWORD (basic auth)
+3. JF_URL + JF_ACCESS_TOKEN (access token)
+
+You can use them in the workflow as follows:
+```yml
+- uses: jfrog/setup-jfrog-cli@v2
+  env:
+    JF_URL: ${{ secrets.JF_URL }}
+    JF_USER: ${{ secrets.JF_USER }}
+    JF_PASSWORD: ${{ secrets.JF_PASSWORD }}
+    # Access token if JF_USER and JF_PASSWORD are not provided
+    # JF_ACCESS_TOKEN: ${{ secrets.JF_ACCESS_TOKEN }}
+- run: |
+    jf rt ping
+    
+    # If both local config(JF_ENV_*) and direct connection details(JF_URL) are provided, the default config will be the local config.
+    # To make the direct connection details config as the default config use the following command:
+    jf c use setup-jfrog-cil-server
```

Comment:
I suggest removing the above paragraph (starting with "If both local config...).
Although it provides valid information, I think it can create confusion.